docs/generating-a-provider.md

```diff
@@ -87,6 +87,22 @@ be quite similar for any other Terraform provider.
        }
        ```
 
+       In a similar way in `cmd/provider/main.go`:
+
+       ```go
+       import (
+           "github.com/crossplane/terrajet/pkg/types/conversion"
+       )
+
+       func main() {
+           ...
+
+           resourceMap := conversion.GetV2ResourceMap(tf.Provider())
+           kingpin.FatalIfError(controller.Setup(mgr, log, rl, setup, ws, pconfig.GetProvider(resourceMap), 1), "Cannot setup Template controllers")
```

Comment:
should we also include how `pconfig` imported?

Reply:
@turkenh `pconfig` is already imported in the template https://github.com/crossplane-contrib/provider-jet-template/blob/main/cmd/provider/main.go#L32 .

Meanwhile `conversion` is not until you comment out https://github.com/crossplane-contrib/provider-jet-template/blob/main/cmd/provider/main.go#L77

So I think we need to highlight only `conversion` import here

Reply:
Alright, sounds good then.